builtin_interfaces/Time stamp
geometry_msgs/Transform frame_to_map
geometry_msgs/Point[] cloud
MadTree tree
float64 weight
int64 seq